.github/workflows/test.yml

```diff
@@ -233,6 +235,18 @@ jobs:
         if: matrix.selenium
         run: echo "PYTEST_ADDOPTS=$PYTEST_ADDOPTS -m selenium" >> "${GITHUB_ENV}"
 
+      - name: Install playwright module
+        run: |
+          pip install playwright
```

Comment:
`playwright` can be added to `dev-requirements.txt`

Reply:
thank you, the changes have been done in setup.py under test section

Reply:
Then this step can be removed.

Reply:
thank you, those rows have been removed, changes have been added